AquaAccount/management/commands/populate.py

```diff
@@ -0,0 +1,329 @@
+            "Danio tęczowy",
+            "Fantom czarny",
+            "Garra panda",
+            "Kirysek czarny",
+            "Kosiarka - Grubowarg syjamski",
+            "Molinezja ostrousta",
+            "Naskalnik Dickfelda",
+            "Neon czerwony",
+            "Paletka - Dyskowiec"
+        ]
+
+        for single_species in fish_species:
+            fish_species_model = Species(name=single_species)
+            fish_species_model.save()
+
+        # create filters
+        aquarium_filters = [
+            "Filtr wewnętrzny do akwarium 10-40l",
+            "Filtr wewnętrzny do akwarium 50-80l",
+            "Filtr wewnętrzny do akwarium 100-160l"
+        ]
+
+        for single_filter in aquarium_filters:
+            filter_model = Filter(type=single_filter)
+            filter_model.save()
+
+        # create heaters
+        aquarium_heaters = [
+            "75W",
+            "100W",
+            "150W",
+            "200W",
+            "25W"
+            "50W"
+            "300W"
+        ]
+
+        for single_heater in aquarium_heaters:
+            heater_model = Heater(power=single_heater)
+            heater_model.save()
+
+        # create pumps
+        aquarium_pumps = [
+            "12W",
+            "20W",
+            "25W",
+            "30W",
+            "40W"
+        ]
+
+        for single_pump in aquarium_pumps:
+            pump_model = Pump(power=single_pump)
+            pump_model.save()
+
+        # create lights
+        aquarium_lights = [
+            "10W LED biała",
+            "10W LED czarna",
+            "16W LED biała 40-60cm",
+            "16W LED czarna 40-60cm",
+            "32W biała 80-100cm",
+            "32W czarna 80-100cm",
+            "10W belka LED biała 50-70cm",
+            "36W belka LED biała 100-120cm"
+        ]
+
+        for single_light in aquarium_lights:
+            light_model = Light(power=single_light)
+            light_model.save()
+
+        while creations < requested:
+            for data in user_generator.generate(requested - creations):
+                if data['email'] in existing_emails:
+                    print(f'User with email {data["email"]} already exists. Skipping')
+                    continue
+                else:
+                    existing_emails.add(data['email'])
+
+                # create user
+                user = User.objects.create_user(
+                    first_name=data['name'],
+                    email=data['email'],
+                    username=data['email'].split('@')[0],
+                    password='default'
+                )
+
+                # create profile for given user
+                profile = user.profile
+                profile.city = 'Warsaw'
+                profile.age = data['age']
+                profile.bio = data['bio']
+                profile.gender = data['gender']
+                profile.save()
+                
+                first_aquarium = Aquarium.objects.create(
```

Comment:
Those models creations could be done in a loop to reduce code duplication

Reply:
naah